tests/DMSDocumentCartControllerTest.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+/**
+ * Class DMSDocumentCartControllerTest
+ */
+class DMSDocumentCartControllerTest extends FunctionalTest
+{
+
```

Comment:
Remove empty line, also line 2. Please either write a useful doc block for the class or remove it.